Support for multi tagging (#43)
amplify-education/serverless-vpc-discovery: CHANGELOG.md

The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).

## [3.0.0] - 2020-12-15
### Added

> - Possibility to specify tag name and tag values for getting subnets and security groups. Also, support for security groups names.  

Reviewer: We always had support for getting security groups by name.

```suggestion
- Support for getting subnets and security groups by any tag key/value
```

### Changed

> - ***Important!*** The `subnetNames` and `securityGroupNames` options have been deprecated but it still will work for a while. The new option are `subnets` and `securityGroups`.

Reviewer: ```suggestion
- ***Important!*** The `subnetNames` and `securityGroupNames` options have been deprecated and will be removed in the next major release. The new options are `subnets` and `securityGroups`.
```
